DuckDuckGo/AppDelegate.swift

```diff
@@ -294,15 +294,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         syncService.scheduler.notifyAppLifecycleEvent()
     }
 
-    func requestSyncIfEnabled() {
-        guard let syncService, syncService.authState == .active else {
-            os_log(.debug, log: OSLog.syncLog, "Sync disabled, not scheduling")
-            return
-        }
```

Comment:
This logic is not needed here because sync scheduler itself doesn't perform any action when sync is not active. I'll add logging to sync scheduler in a separate change to BSK.